packages/axios-extension/src/base/odata-request-error.ts

```diff
@@ -1,3 +1,15 @@
+import type { AxiosError } from 'axios';
+
+/**
+ * Casts an unknown error to an AxiosError.
+ *
+ * @param e unknown error
+ * @returns exception casted to AxiosError if it is one
+ */
+export function isAxiosError(e: unknown): e is AxiosError {
```

Comment:
The `isAxiosError()` function is exposed by `axios@1.x`. Guess this will can be replaced when we upgrade?

Reply:
Wow, I missed that there is a version `1.x` of `axios`. However, my implementation is better for `TypeScript` usage :).
Let's re-evaluate it when we switch to `axios@1`